log_merger.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pandas as pd
+
+
+class LogMerger:
+    def __init__(self,
+                 infile: str,
+                 msg_types: list[str],
+                 max_msgs: int,
+                 max_rows: int,
+                 verbose: bool):
+        self.infile = infile
+        self.prefix = infile.split('.')[0]
+        self.msg_types = msg_types
+        self.max_msgs = max_msgs
+        self.max_rows = max_rows
+        self.verbose = verbose
+        self.tables = None
+
+    def outfile(self, suffix: str = '', ext: str = '.csv'):
```

Comment:
On line 13, infile is a variable name. On line 21 outfile is a method name. That could be confusing.

Reply:
Roger, I suppose the function could be called 'get_outfile_name'.